Animated-Glassmorphism-Vanila-Js/README.md

```diff
@@ -0,0 +1,16 @@
+<h1>Simon Game</h1>
+
+<h2>Overview</h2>
+
+<p>This application is made for beginners.User can use this web app to play Simon Game.</p>
+
+<h3>Tech Stack</h3>
+
+![HTML](https://img.shields.io/badge/html5%20-%23E34F26.svg?&style=for-the-badge&logo=html5&logoColor=white)
+![CSS](https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white)
+![JS](https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E)
+
+## Screenshots
+![Demo1](https://raw.githubusercontent.com/CoderSahil128/WebDev-ProjectKart/master/Animated-Glassmorphism-Vanila-Js/Demo1.jpeg)
+![Demo2](https://raw.githubusercontent.com/CoderSahil128/WebDev-ProjectKart/master/Animated-Glassmorphism-Vanila-Js/Demo2.jpeg)
+![Demo3](https://raw.githubusercontent.com/CoderSahil128/WebDev-ProjectKart/master/Animated-Glassmorphism-Vanila-Js/Demo3.jpeg)
```

Comment:
Giving Relative path in the same Repository will be a better option, If you are using Path from your Repository, then If someday you privates/delete your repository, then these Images will be no more accessible